esmvaltool/utils/batch-jobs/generate.py

```diff
@@ -272,8 +290,15 @@ def generate_submit():
             else:
                 file.write(f'esmvaltool run --config_file '
                            f'{str(config_file)} {str(recipe)}')
-            if recipe.stem in ONE_TASK_RECIPES:
-                file.write(' --max_parallel_tasks=1')
+            # max_parallel_tasks
+            # Default
+            if not MAX_PARALLEL_TASKS.get(recipe.stem, None):
+                file.write(f' --max_parallel_tasks='
+                           f'{default_max_parallel_tasks}\n')
+            # Special requirements
+            else:
+                file.write(f' --max_parallel_tasks='
+                           f'{MAX_PARALLEL_TASKS[recipe.stem]}\n')
```

Comment:
@remi-kazeroni It may be possible to write this without the `if` statement. That would make the code easier to read. Would the following also work?
```suggestion
            max_parallel_tasks = MAX_PARALLEL_TASKS.get(
                recipe.stem,
                default_max_parallel_tasks,
            )
            file.write(f' --max_parallel_tasks={max_parallel_tasks}\n'
```

Reply:
Sure, done in 3e7de06. Works better like this